docs/source/compression/compression_config_list.rst

```diff
@@ -1,11 +1,190 @@
 Compression Config Specification
 ================================
 
+Common Keys in Config
+---------------------
+
+op_types
+^^^^^^^^
+
+A list of type names of classes that inherit from ``torch.nn.Module``.
+Only module types in this list can be selected to be compressed.
+If this key is not set, all module types can be selected.
+
+op_names
+^^^^^^^^
+
+A list of fully-qualified name of modules that will be compressed.
+If the name referenced module is not existed in the model, it will be ignored.
+
+op_names_re
+^^^^^^^^^^^
+
+A list of regular expressions for matching module names.
+The matched modules will be selected to be compressed.
+
+exclude_op_types
+^^^^^^^^^^^^^^^^
+
+A list of type names of classes that inherit from ``torch.nn.Module``.
+Module types in this list can not be selected to be compressed.
```

Comment:
-> "The module types in this list are excluded from compression."

Reply:
fixed